rules/follow-naming-conventions-for-tests-and-test-projects/rule.md

```diff
@@ -61,21 +63,31 @@ Remember that descriptive names are useful – but the choice of name is not the
 :::
 
 ```cs
-[Method/PropertyName]_Should_[ExpectedBehavior]_When_[StateUnderTest]
+{SUT}_{Condition}_{Should}
 ```
+**Figure: The naming convention is effective – it encourages developers to clearly define the expected result upfront without requiring too much verbosity** 
 
-**Figure: The "should" naming convention is effective – it encourages developers to clearly define the expected result upfront without requiring too much verbosity** 
+Note: 'SUT' is a System Under Test.
 
-The following test names use the "should" naming convention:
+Think of this as 3 parts, separated by underscores:
```

Comment:
I'm not sure that Given/When/Then aligns to your naming convention (at least from the descriptions you've used).

For example, your naming places SUT first, but given/when/then places SUT second.

We're also changing the naming of the SUT.  The original naming convention indiciated this should be a method/class name.  But using GIVEN this could be a scenario.  i.e.  "GIVEN an account with a balance of $100..."

The more I think about this, the more I'm unsure if Given/When/Then is a good fit to describe the naming convention.

Reply:
@danielmackay I made the same statement

Using your Given, When, Then statement

From 
Checkout_WithCountryAsAustralia_ShouldAdd10PercentTax

To
WithCountryAsAustralia_Checkout_ShouldAdd10PercentTax

Reply:
When using GWT convention I would normally write the test case as:

`GivenCountryAsAustralia_WhenCheckout_ThenAdd10PercentTax`